package.json

```diff
@@ -11,7 +11,8 @@
     "clean:allowlist": "node src/clean-config.js allowlist",
     "add:blacklist": "node src/blacklist.js",
     "add:blocklist": "node src/add-hosts.js src/config.json blocklist",
-    "add:allowlist": "node src/add-hosts.js src/config.json allowlist"
+    "add:allowlist": "node src/add-hosts.js src/config.json allowlist",
+    "prepare": "husky install"
```

Comment:
(could) use pre-commit package if the functionality suffices - it's much more lightweight